### DISCUSSION TITLE
queryTags & Filters - вывод в режиме resourceMode

### DISCUSSION THREAD

**Discussion by IgorPetrovPRO** [moonshine-software/moonshine#374] [Ideas]
Если у ресурса задать queryTags то они выводятся только на основной странице https://disk.yandex.ru/d/NxQ4tp3-tbENDA но при этом не выводятся в карточке когда смотришь в resourceMode https://disk.yandex.ru/d/jvxh8IVZHMM-BQ 
Было бы классно если можно было бы и там вывести и например добавить параметр, что бы определять где выводить, а где нет.

Аналогично и с Filters

**Comment by lee-to** [comment]
Добавим

**Comment by lee-to** [comment]
https://github.com/moonshine-software/moonshine/pull/403
пока без фильтров но с поиском
